Makefile

```diff
@@ -1,24 +1,33 @@
+# Documentation
 docs/tuist/preview:
 	./make/tasks/docs/tuist/preview.sh
 docs/tuist/build:
 	./make/tasks/docs/tuist/build.sh
-edit:
-	./make/tasks/edit.sh
-fetch:
-	./make/tasks/fetch.sh $(ARGS)
-generate:
-	./make/tasks/generate.sh $(ARGS)
-generate/with-cloud:
-	TUIST_INCLUDE_TUIST_CLOUD=1 ./make/tasks/generate.sh $(ARGS)
-run:
-	./make/tasks/run.sh $(ARGS)
-generate/cloud-openapi-code:
-	./make/tasks/generate/cloud-openapi-code.sh
-lint-fix:
-	./make/tasks/lint-fix.sh
-lint:
-	./make/tasks/lint.sh
-lint/lockfiles:
-	./make/tasks/lint/lockfiles.sh
-pull/cloud:
-	git submodule update --init submodules/TuistCloud
\ No newline at end of file
+
+# Tuist
+tuist/edit:
+	./make/tasks/tuist/edit.sh
+tuist/fetch:
+	./make/tasks/tuist/fetch.sh $(ARGS)
+tuist/generate:
+	./make/tasks/tuist/generate.sh $(ARGS)
+tuist/generate/with-cloud:
+	TUIST_INCLUDE_TUIST_CLOUD=1 ./make/tasks/tuist/generate.sh $(ARGS)
+tuist/run:
+	./make/tasks/tuist/run.sh $(ARGS)
+	
+# Cloud
+cloud/up:
```

Comment:
Is it possible to read an env variable from `Package.swift`, so this command doesn't cause a git change there when flipping the `includeTuistCloud` flag? And we can definitely do that for `Project.swift`.

Reply:
It is possible when invoking the SPM with `--disable-sandbox`, but that only works when interacting with the package from the terminal. If you open the package in Xcode, Xcode will evaluate the packages with the sandbox enabled, and it fails to resolve.

Reply:
Alright, I iterated on the approach:
- `TUIST_INCLUDE_TUIST_CLOUD`: Is used to control including or not the sources of Tuist Cloud.
- There's now only one Make task, `make workspace/up` that decides what to do based on the value of `TUIST_INCLUDE_TUIST_CLOUD`.
- When opening the `Package.swift` directly in Xcode, Xcode evaluates the `Package.swift` in a sandboxed environment, so developers working with the TuistCloud sources won't be able to control the opt-in via environment variables and instead, they'll have to set the value directly to "include = true".